README.md

```diff
@@ -10,127 +10,99 @@
-
-- Clone/Download the repository
-  - To clone it and get the updates you can definitely use the command
-    `git clone`
-- Create a discord bot [here](https://discord.com/developers/applications)
-- Get your bot token
+- A basic knowledge of Python.
+- A Discord bot application. You can create one [here](https://discord.com/developers/applications).
 - Invite your bot on servers using the following invite:
   https://discord.com/oauth2/authorize?&client_id=YOUR_APPLICATION_ID_HERE&scope=bot+applications.commands&permissions=PERMISSIONS (
   Replace `YOUR_APPLICATION_ID_HERE` with the application ID and replace `PERMISSIONS` with the required permissions
   your bot needs that it can be get at the bottom of a this
   page https://discord.com/developers/applications/YOUR_APPLICATION_ID_HERE/bot)
 
-## How to set up
+## Setup
 
-To set up the bot it was made as simple as possible.
+1. **Configure `config.json`**:
+   - Set your bot's prefix and invite link in the `config.json` file.
+   - Here is an explanation of what everything is:
+    
+      
+      | Variable                  | What it is                                     |
+      | ------------------------- | ---------------------------------------------- |
+      | YOUR_BOT_PREFIX_HERE      | The prefix you want to use for normal commands |
+      | YOUR_BOT_INVITE_LINK_HERE | The link to invite the bot                     |
+      
 
-### `config.json` file
+2. **Set the Bot Token**:
+   - Create an environment variable named `TOKEN` or edit the `.env` file with your bot token.
+   - To set up the token you will have to either make use of the [`.env.example`](.env.example) file, either copy or rename it to `.env` and replace `YOUR_BOT_TOKEN_HERE` with your bot's token.
 
-There is [`config.json`](config.json) file where you can put the
-needed things to edit.
+3. **Install Requirements**:
+   - Run `python -m pip install -r requirements.txt` to install the required packages.
 
-Here is an explanation of what everything is:
+4. **Start the Bot**:
+   - Run `python bot.py` to launch your bot.
 
-| Variable                  | What it is                                     |
-| ------------------------- | ---------------------------------------------- |
-| YOUR_BOT_PREFIX_HERE      | The prefix you want to use for normal commands |
-| YOUR_BOT_INVITE_LINK_HERE | The link to invite the bot                     |
+## Support
 
-### `.env` file
+If you need help or have questions, feel free to join our [Discord server](https://discord.com/invite/mTBrXyWxAF) for assistance.
 
-To set up the token you will have to either make use of the [`.env.example`](.env.example) file, either copy or rename it to `.env` and replace `YOUR_BOT_TOKEN_HERE` with your bot's token.
 
-Alternatively you can simply create an environment variable named `TOKEN`.
+## Versioning
 
-## How to start
+We use [SemVer](https://semver.org/) for versioning. Check the repository's tags for available versions.
```

Comment:
Here as well, I've made the mistake before to use "we" - to formulate it better I'd say something along the lines
> This repository uses ...